lib/public/components/Log/replyButton.js

```diff
@@ -25,7 +25,7 @@ export const replyButton = (log) => frontLink(
     { parentLogId: log.id },
     {
         id: `reply-to-${log.id}`,
+        style: 'margin-right: 5px',
```

Comment:
(almost) never use inline styles, and absolutely never use hard-coded pixels dimensions: it prevent UI from having an homogenous display, it prevent anyone to properly zoom in and so on.
I know it's legacy code but you touched it so now you're cursed and you have to clean it.
You have utility classes such as `m1`, `m2`, `m3` to apply pre-defined margins.